packit_service/worker/build/babysit.py

```diff
@@ -94,7 +93,11 @@ def check_copr_build(build_id: int) -> bool:
         )
 
         for job_config in job_configs:
+            event_dict = event.get_dict()
             CoprBuildEndHandler(
-                ServiceConfig.get_service_config(), job_config=job_config, event=event,
+                package_config=event.package_config,
+                job_config=job_config,
+                data=EventData.from_event_dict(event_dict),
+                event=event_dict,
```

Comment:
Can we rename `event` to `event_dict` to make it clear? OR encapsulate the raw `event_dict` to `EventData`?

Reply:
going to change it

Reply:
Thanks!